packages/web/src/services/audius-backend/BuyAudio.ts

```diff
@@ -42,6 +48,15 @@ export const getSolanaConnection = async () => {
   return libs().solanaWeb3Manager.connection as Connection
 }
 
+export const getRootAccountRentExemptionMinimum = async () => {
+  await waitForLibsInit()
+  const connection = await getSolanaConnection()
+  return await connection.getMinimumBalanceForRentExemption(
+    ROOT_ACCOUNT_SIZE,
+    'finalized'
```

Comment:
this could probably be 'processed' (maybe would be faster? not sure)

Reply:
good point!